client/my-sites/email/email-management/home/email-list-inactive.jsx

```diff
@@ -1,18 +1,50 @@
 import { Button, Card } from '@automattic/components';
 import { localize } from 'i18n-calypso';
 import React from 'react';
+import emailIllustration from 'calypso/assets/images/email-providers/email-illustration.svg';
+import PromoCard from 'calypso/components/promo-section/promo-card';
 import SectionHeader from 'calypso/components/section-header';
 import { canCurrentUserAddEmail } from 'calypso/lib/domains';
-import { emailManagementPurchaseNewEmailAccount } from 'calypso/my-sites/email/paths';
+import {
+	emailManagementPurchaseNewEmailAccount,
+	isUnderEmailManagementInbox,
+} from 'calypso/my-sites/email/paths';
 
 class EmailListInactive extends React.Component {
+	getMainHeader() {
+		const { translate } = this.props;
+		const image = {
+			path: emailIllustration,
+			align: 'right',
+		};
+
+		return (
+			<>
+				<PromoCard title={ translate( 'Pick a domain to get started' ) } image={ image }>
+					<p>
+						{ translate(
+							'Pick a domain from your available domains below to add an email solution.'
```

Comment:
How about this simpler wording as well? cc @saygunnyc 
```suggestion
							'Pick one of your domains below to add an email solution.'
```

Reply:
In the meanwhile, I have added your suggestion, if Saygun thinks that it's better the other wording I'll change it :)